test/framework/kogitomgmtconsole.go

```diff
@@ -15,100 +15,50 @@
 package framework
 
 import (
-	"fmt"
-	"strconv"
-
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	"github.com/kiegroup/kogito-cloud-operator/test/config"
-	apps "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// InstallKogitoManagementConsole deploy the Kogito Management Console
+// InstallKogitoManagementConsole returns the installation service for the Kogito Management Console
```

Comment:
this method does return only an error ;)

Reply:
Changed